### DISCUSSION TITLE
post/six-months-in-Tencent

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#24] [General]
# post/six-months-in-Tencent

✨ 本文谨以纪念我此刻 21 岁的青春（图：深圳南山公园祝寿亭夜景）  今年（2024），鄙人有幸在腾讯 CSIG 作为 Golang 后端开发实习了六个月 我只能说，我在深圳腾讯的这六个月，是我目前人生中最美好的时光 过去从没有过，将来也可有能不会再有 我在此记录下这六个月中的点点滴滴，作为将

https://nickxu.me/post/six-months-in-Tencent

**Comment by x1uc** [comment]
How can we say goodbay , as we say hello.

**Comment by dubinbin** [comment]
虽然祛魅，腾讯关怀和文化确实是不错的，虽然有各种那样的问题，但仍然觉得是一家好公司、好神会大学。 离开很久了，但腾讯人仍旧是我比较喜欢的tag。

**Comment by Punny520** [comment]
很棒的博客，感谢分享

**Comment by blacksheepaul** [comment]
写的很棒，申请季祝你一切顺利
